e2e/tests/messageJudge_test.js

```diff
@@ -4,6 +4,8 @@ const mandatoryWithAdditionalApplicationsBundle = require('../fixtures/caseData/
 let caseId;
 let message = 'Some note';
 let reply = 'This is a reply';
+const messageHistoryInitial = config.ctscEmail + ' - ' +  message;
+const messageHistoryReply = config.ctscEmail + ' - ' +  message + '\n \n' + config.judicaryUser.email + ' - ' + reply;
```

Comment:
I would define this as `messageHistoryInitial + ...` and also it would probs be nicer if you used template strings here (would have to make sure it is syntactically valid)
```suggestion
const messageHistoryInitial = `${config.ctscEmail} - ${message}`;
const messageHistoryReply = `${messageHistoryInitial}\n \n${config.judicaryUser.email} - ${reply}`;
```
Also is there a reason why there is a space between the 2 `\n`?

Reply:
The `buildMessageHistory` function in `MessageJudgeService.java` has been adding a space between the two `\n` so the test will fail without it